src/pages/patientView/structuralVariant/StructuralVariantTableWrapper.tsx

```diff
@@ -316,9 +318,67 @@ export default class StructuralVariantTableWrapper extends React.Component<
                         this.props.store.uniqueSampleKeyToTumorType
                     );
                 },
+                visible:
+                    getServerConfig().show_oncokb &&
+                    this.props.store.structuralVariantOncoKbData.result
```

Comment:
i think that _.isEmpty might cover ALL of these conditions if you use optional chaining.

 if so then apply that here and the other visible tests.